src/environment.jl

```diff
@@ -0,0 +1,25 @@
+
+Base.@kwdef struct EnvironmentalPars{A,P,E,L,F}
```

Comment:
we also should think about what it will mean if we want these to change through a simulation, e.g. animal moving in and out of water behaviourally, changing elevation as it flies, changing gas concentrations under ground, etc.